Clarify Peer Cert management docs (#16764)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

1. Click **Create**.

> 1. Switch to the active instance, and then switch to **Instance Configuration**. Click **Add Instance**, enter the new standby instance's IP address or hostname, including the HTTP or HTTPS protocol prefix and port if you are not using the default of 80 or 443. If the HTTPS protocol is enabled, you can use the **Manage Peer Certificates** dialog to set up a root CA certificate of all the instances. Make sure that on the active instance we have added root CA certs for the **active as well as all the standby**. 

Reviewer: do these caveats also not apply to the equivalent step under Configure the active instance?

also, the wording is unclear - do you add one certificate for all instances ("set up a root CA certificate for all the instances"), or one per instance ("added root CA certs for the **active as well as all the standby**")?

Author: If the instances are using self signed certificate then it will be separate certificate.  But many of our customers use enterprise CA that is common across their org. So all the instances will have same root CA certificate. You can still add it for each instance.
But describing all of this is getting into a tutorial of what is a root CA. So I'm not sure we want to give that much of detail.

> 1. Switch to the active instance, and then switch to **Instance Configuration**. Click **Add Instance**, enter the new standby instance's IP address or hostname, including the HTTP or HTTPS protocol prefix and port if you are not using the default of 80 or 443. If the HTTPS protocol is enabled, you can use the **Manage Peer Certificates** dialog to set up a root CA certificate of all the instances. Make sure that on the active instance we have added root CA certs for the **active as well as all the standby**. 

Reviewer: ```suggestion
1. Switch to the active instance, and then switch to **Instance Configuration**. Click **Add Instance**, enter the new standby instance's IP address or hostname, including the HTTP or HTTPS protocol prefix and port if you are not using the default of 80 or 443.

1. If the HTTPS protocol is enabled, you can use the **Manage Peer Certificates** dialog to set up a root CA certificate for all the instances. Make sure that on the active instance you added root CA certificates for the active as well as all the standbys. 
```
